### PR TITLE
LUN-365: Header selection screen broken

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

BottomTabBar fix only testable on ios with a device that has notches. (f.e. iPhone 13 max)

### CODE REVIEW

File: src/navigation/BottomTabNavigator.tsx [hunk 1]
```diff
@@ -13,78 +13,78 @@ const Navigator = createBottomTabNavigator<RoutesParams>()
...
+    <Navigator.Navigator
+      screenOptions={{
+        headerShown: false,
+        tabBarActiveTintColor: theme.colors.background,
+        tabBarStyle: {
+          backgroundColor: theme.colors.primary,
+          minHeight: wp('14%'),
+          paddingBottom: insets.bottom,
+          // TODO LUN-132: Delete
+          display: 'none'
```
Comment (f1sh1918) [inline src/navigation/BottomTabNavigator.tsx:28]: remove this line to test

**Review by SteffiStuffel — APPROVED** [review]
Could not test, as browserstack is down, but looks good to me.

**Comment by steffenkleinle** [comment]
could you update this to the current state of main and resolve merge conflicts? Than no adjustments are needed to test this :)

**Comment by f1sh1918** [comment]
> could you update this to the current state of main and resolve merge conflicts? Than no adjustments are needed to test this :)

done :)

**Review by steffenkleinle — APPROVED** [review]
Was not able to test this on browserstack as the app kept freezing all the time :/

**Pull request merged**